src/merge.js

```diff
@@ -243,8 +243,9 @@ class Merge {
         doc.creationDate = file.creationDate
         if (this.sameBinary(file, doc)) {
           if (doc.size == null) { doc.size = file.size }
-          if (doc.class == null) { doc.class = file.class }
-          if (doc.mime == null) { doc.mime = file.mime }
+
+          doc.class = file.class
+          doc.mime = file.mime
```

Comment:
@sebn I don't see how this will work. If local and remote see the same file with different mime-type, the doc will loop with one time the mime-type from local and next time the mime-type from remote, no?